my-react-app/src/components/LoginPage.js

```diff
@@ -0,0 +1,39 @@
+import { Auth } from 'aws-amplify';
```

Comment:
Consider using a singleton pattern, the Singleton pattern is a design pattern used to restrict the instantiation of a class to a single instance and provide a global point of access to that instance. In the context of integrating AWS Amplify and Cognito, it can be helpful for the amplify configuration, and making service instances consistent.